readme.md

```diff
@@ -406,7 +406,7 @@
 - [Incident Response](https://github.com/meirwah/awesome-incident-response)
 - [Vehicle Security and Car Hacking](https://github.com/jaredmichaelsmith/awesome-vehicle-security)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security) - Security of web apps & services.
-- [Lockpicking](https://github.com/meitar/awesome-lockpicking)
+- [Lockpicking](https://github.com/meitar/awesome-lockpicking) - Resources relating to the security and compromise of locks, safes, and keys.
```

Comment:
@sindresorhus I've changed this to "Techniques and tools for opening a lock without the key and without damaging or destroying the lock or its door." and pushed an updated commit.